devfiles/nodejs-react/meta.yaml

```diff
@@ -0,0 +1,6 @@
+---
+displayName: React web application
```

Comment:
For consistency with other stacks I would rather rename it `NodeJS React Web Application`